src/test/java/org/cyclonedx/BomXmlGeneratorTest.java

```diff
@@ -95,6 +95,19 @@ public void schema11WithDependencyGraphGenerationTest() throws Exception {
         Assert.assertTrue(parser.isValid(file, CycloneDxSchema.Version.VERSION_11));
     }
 
+    @Test
```

Comment:
arent tests missing here? AFAIK we added a lot for vulnerabilities, only see for schema 11, also it'd be good to check the real values , I think we did it the first time, because the valid was working but we got the enums wrong in the outcome